tests/integration-e2e/test_connection.py

```diff
@@ -6,6 +6,7 @@
 
 SIM_DIR = Path(__file__).parent.parent.absolute() / "simulations" / "v5_sonata"
 CONFIG_FILE_MINI = SIM_DIR / "simulation_config_mini.json"
+CONFIG_FILE_ERR = SIM_DIR / "simulation_config_err.json"
```

Comment:
Can we create a temp config file using the basic data `sonata_config()` from `conftest.py` ? Maybe we can set fixtures like in `test_mulitpop.py` ? 
```
@pytest.fixture
def sonata_config_file(sonata_config, extra_config):
```

Reply:
done